src/app/components/candidate-request/candidate-request.component.scss

```diff
@@ -1,2 +1,71 @@
 @import '../../shared/constants/colors.scss';
 @import '../../shared/constants/otherStyles.scss';
+
+form {
+  padding: 5px 15px;
+}
+
+.sandbox > * {
```

Comment:
* selector is not a good idea, it affects performance. You can leave it like this for now, but next time use classes or something else.

Reply:
ok, thx